packages/core/src/modules/event/attendance-service.ts

```diff
@@ -1,6 +1,6 @@
 import { type AttendanceId, type Attendee, type EventId, type UserId } from "@dotkomonline/types"
 import { type AttendanceRepository } from "./attendance-repository"
-import { AttendeeNotFoundError } from "./attendee-error"
+import { EventNotFoundError } from "./event-error"
```

Comment:
Can you explain why EventNotFoundError is used here? The idea is to encode the specific cases. Any missing entity in the Event domain should not throw EventNotFoundError, but the specific type that is missing